sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessErrorEventArgs.cs

```diff
@@ -37,12 +40,12 @@ public ProcessErrorEventArgs(Exception exception, ExceptionReceivedEventArgsActi
         public ExceptionReceivedEventArgsAction Action { get; }
 
         /// <summary>The namespace name used when this exception occurred.</summary>
-        public string Endpoint { get; }
+        public string FullyQualifiedNamespace { get; }
```

Comment:
This is a namespace name. What's a fully qualified namespace? There's an FQDN, but doesn't seem like what you've meant.

Reply:
So this would be the host name for the account, e.g. {yournamespace}.servicebus.windows.net

Reply:
If this is the hostname for the account, then it's an FQDN (FullyQualifiedDomainName).
What I'm trying to say is that there's no namespace and fully qualified namespace. There are namespace and FQDN. Correct me if I'm wrong, @nemakam.